shell/platform/android/android_native_window.cc

```diff
@@ -6,16 +6,7 @@
 
 namespace shell {
 
-AndroidNativeWindow::AndroidNativeWindow(Handle window) : window_(window) {
-  if (window_ != nullptr) {
-    ANativeWindow_acquire(window_);
-  }
-}
-
-AndroidNativeWindow::AndroidNativeWindow(AndroidNativeWindow&& o)
-    : window_(o.window_) {
-  o.window_ = nullptr;
-}
+AndroidNativeWindow::AndroidNativeWindow(Handle window) : window_(window) {}
```

Comment:
There is a similar class used internally by skia that assumes that ownership of the handle is being passed in. As I was typing it again, I had to do a double take on whether the ownership rules were those following by Skia or by us. I just modified our behavior to match those of the class in skia for consistency.